enhancements/installer/aws-permissions-check-bypass.md

```diff
@@ -105,43 +112,63 @@ in-cluster behavior of cloud-credential-operator.
+	credentialsModeManual credentialsMode = "manual"
 )
 
 type InstallConfig struct {
-	// ForceCredentialsMode instructs the installer to not attempt to query
-	// the cloud permissions before attempting installation. It also passes
-	// down the desired credentials mode to the cloud-credential-operator
-	// so that it too does not attempt to query permissions.
+	// CredentialsMode is used to explicitly set the mode with which
+	// CredentialsRequests are satisfied.
+	//
+	// If this field is set, then the installer will not attemp to query the
+	// cloud permissions before attempting installation. If the field is not
+	// set or empty, then the installer will perform its normal verification
+	// that the credentials provided are sufficient to perform an
+	// installation.
+	//
+	// There are three possible values for this field, but the valid values
+	// are dependent upon the platform being used.
+	// "mint": create new credentials with a subset of the overall
+	// permissions for each CredentialsRequest
+	// "passthrough": copy the credentials with all of the overall
+	// permissions for each CredentialsRequest
+	// "manual": CredentialsRequests must be handles manually by the user.
+	//
+	// For each of the following platforms, the field can be set to the
+	// specified values. For all other platforms, the field must not be set.
+	// AWS: "mint", "passthrough", "manual"
+	// Azure: "mint", "passthrough"
+	// GCP: "mint", "passthrough"
 	// +optional
-	ForceCredentialsMode cloudCredentialsMode `json:"forceCredentialsMode,omitempty"`
+	CredentialsMode credentialsMode `json:"credentialsMode,omitempty"`
 }
 ```
 
-The installer will then take the value defined in the `forceCredentialsMode`
-field to populate the config for the cloud-credential-operator so that CCO knows
-what mode it should be forced into.
+The installer will make available the user's install-config as a ConfigMap that
+the cloud-credential-operator can then use to affect CCO runtime behavior.
 
 ### cloud-credential-operator
 Formalize the constants in cloud-credential-operator repo to define the
 acceptable credentials (matching the definitions in the installer):
 ```
-type CloudCredentialsMode string
+type CredentialsMode string
 
 const (
-	// MintMode is the annotation on the cluster's cloud credentials to
-	// indicate that CCO should be creating users for each
+	// ModeMint indicates that CCO should be creating users for each
 	// CredentialsRequest.
-	MintMode CloudCredentialsMode = "mint"
+	ModeMint CredentialsMode = "mint"
 
-	// PassthroughMode is the annotation on the cluster's cloud credentials
-	// to indicate that CCO should just copy over the cluster's cloud
+	// ModePassthrough indicates that CCO should just copy over the cluster's cloud
 	// credentials for each CredentialsRequest.
-	PassthroughMode CloudCredentialsMode = "passthrough"
+	ModePassthrough CredentialsMode = "passthrough"
+
+	// ModeManual indicates that CCO should not process CredentialsRequests.
```

Comment:
What does it mean to `process` CredentialRequests ?

- no changes to the object at all, like adding status that it is in manual mode.. or,
- no iam like actions for the object,
- what about derivatives of the credential requests like the secret that is created in the namespace of the target..

I think we need to think about what it means to be manual and maybe describe that a little more here.

Reply:
@abhinavdahiya as the CCO code currently functions (or is intended to function), it means that it will report metrics but never update the status of a CredentialsRequest (no provisioned: True, no conditions, nothing)
It means the user is on the hook to create the Secrets for the components that issued the CredentialsRequest resource.
We originally called it "disabled" mode b/c we thought of it as truly completely disabled, but over time there has been a desire to be more helpful in communicating status (like for upgrades) even when CCO isn't responsible for creating the Secrets with the credentials in them.

I'll update the PR with the details